.golangci.yml

```diff
@@ -15,11 +15,19 @@ linters-settings:
     # put imports beginning with prefix after 3rd-party packages;
     local-prefixes: github.com/elastic/cloud-on-k8s
 
+  gosec:
+    excludes:
+      - G115 # potential integer overflow when converting between integer types
```

Comment:
This one is interesting, I had a quick look and there are indeed some cases where conversion is no needed or maybe dangerous. I would maybe create a follow-up issue with the `good first issue` label?

Reply:
Good call!
- https://github.com/elastic/cloud-on-k8s/issues/8076